src/components/QueryEditor.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button, QueryField } from '@grafana/ui';
+import { QueryEditorProps } from '@grafana/data';
+import { DataSource } from '../datasource';
+import { SumoQuery } from '../types/metricsApi.types';
+
+type Props = QueryEditorProps<DataSource, SumoQuery>;
+
+export function QueryEditor({ query, onChange, onRunQuery }: Props) {
+  const onQueryTextChange = (changedQueryText: string) => {
+    onChange({ ...query, queryText: changedQueryText });
+  };
+
+  const { queryText } = query;
+
+  return (
+    <div style={{ display: 'flex', gap: '1rem' }}>
```

Comment:
Don't we have any possibility to use CSS here (SCSS/LESS/CSS Modules/etc)?

Reply:
we don't have any CSS here, so I think it simplier to keep it this way